frog/imports/ui/Logo/index.js

```diff
@@ -5,43 +5,26 @@ import { makeStyles } from '@material-ui/core';
+      fill="#2C3C3E"
+    />
+    <circle cx="133" cy="133" r="133" fill="#22A6B3" />
+    <circle cx="180" cy="90" r="10" fill="white" />
+    <path
+      d="M60 172.542C60 167.263 63.6761 162.696 68.8336 161.568L126.365 148.983C133.371 147.45 140 152.786 140 159.958V159.958C140 165.237 136.324 169.804 131.166 170.932L73.6351 183.517C66.6289 185.05 60 179.714 60 172.542V172.542Z"
+      fill="white"
+    />
+    <path
+      d="M65 140.06C65 134.772 68.6865 130.2 73.8542 129.078L191.878 103.457C198.882 101.937 205.5 107.273 205.5 114.44V114.44C205.5 119.728 201.813 124.3 196.646 125.422L78.6223 151.043C71.6184 152.563 65 147.227 65 140.06V140.06Z"
+      fill="white"
+    />
```

Comment:
Why did you change the logo? :frog:

Reply:
Just experimenting. I was trying to make a frog symbol from the letter F. It felt more neat. Feel free to revert back though!

Reply:
It's cool. Keep it this way :)